generator/src/main/java/com/scottlogic/deg/generator/restrictions/ReductiveRowSpec.java

```diff
@@ -14,14 +14,17 @@
 import java.util.stream.Stream;
 
 public class ReductiveRowSpec extends RowSpec {
-
     private final Field lastFixedField;
 
     public ReductiveRowSpec(ProfileFields fields, Map<Field, FieldSpec> fieldToFieldSpec, Field lastFixedField) {
         super(fields, fieldToFieldSpec);
         this.lastFixedField = lastFixedField;
     }
 
+    public Field getLastFixedField() {
```

Comment:
Why has this field been made public?

Reply:
To support tests. We may be able to remove this by adding an equals() for the type